tests/integration/REST_API/Controllers/SettingsTest.php

```diff
@@ -162,6 +162,64 @@ public function test_get_item_not_found() {
 	}
 
 
+	/** @see Settings::update_item() */
+	public function test_update_item() {
+
+		$settings   = $this->get_settings_instance();
+		$controller = new Settings( $settings );
+
+		$setting = $settings->get_setting( 'test_one' );
+		$request = new WP_REST_Request( 'POST', "/wc/v3/{$settings->get_id()}/settings/{$setting->get_id()}" );
+
+		$request->set_header( 'content-type', 'application/json' );
+		$request->set_url_params( [ 'id' => $setting->get_id() ] );
+		$request->set_body( json_encode( [ 'value' => 'a' ] ) );
+
+		$response = $controller->update_item( $request );
+
+		$this->assertInstanceOf( WP_REST_Response::class, $response );
+		$this->assertSame( 200, $response->get_status() );
+
+		$this->assert_item_matches_setting( $response->get_data(), $setting );
```

Comment:
Should we maybe also test if the value was really updated in the database, by getting it again?

Reply:
Makes sense. I retrieved and compared the value stored in the associated option in 623373c. Also added an assert to confirm the returned value matches the one we sent.